layouts/partials/head.html

```diff
@@ -26,8 +27,4 @@
 
 	{{ "<!-- Google Fonts -->" | safeHTML }}
 	<link href="//fonts.googleapis.com/css?family=Source+Sans+Pro:400,700,700italic,400italic" rel="stylesheet" type="text/css">
-
-	<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.6.0/styles/default.min.css">
```

Comment:
You wrote in the commit X that we will use `chroma`. Do you mean [alecthomas/chroma](https://github.com/alecthomas/chroma)?
If yes, how can we use it? Is it built in, in Hugo? Or how does it work?
Because in the commit i only removed highlight.js.

Reply:
> Is it built in, in Hugo?

yes, it is builtin on hugo.

just need to set a style -> https://github.com/caarlos0/carlosbecker.com/blob/master/config.toml#L9-L10

https://gohugo.io/content-management/syntax-highlighting/